.storybook/storybook-styles.css

```diff
@@ -1,18 +1,25 @@
 .docs-story > div:first-child {
   min-height: 500px;
-  background-color: #212425;
+  background-color: #F8F8F8;
```

Comment:
Again, not important, why?

Reply:
It was fixing something but seems it's not anymore. removed